src/components/dashboard/Dashboard.js

```diff
@@ -633,17 +633,16 @@ const Dashboard = props => {
     const minScrollRequiredISH = headerLarge ? minScrollRequired : minScrollRequired * 2
     const scrollPositionGap = headerLarge ? 0 : minScrollRequired
     const newsCondition = activeTab === FeedCategories.News && feedRef.current.length > 3
-    const isFeedSizeEnough = feedRef.current.length > 10 || newsCondition
-
+    const isFeedSizeEnough = feedRef.current.length > 8 || newsCondition
     return { minScrollRequiredISH, scrollPositionGap, isFeedSizeEnough }
-  }, [headerLarge, activeTab])
+  }, [headerLarge, activeTab, feedRef.current.length])
```

Comment:
ref shouldn't be a dep

Reply:
Without it scroll data wasn't updated than length was changed

Reply:
then add `feeds` to the deps and use feeds.length inside

Reply:
or even add feed.length as the dep

Reply:
@roma-hladilka ^^^